source/openstack/details.rst

```diff
@@ -0,0 +1,115 @@
+.. _openstack-details:
+
+OpenStack Details
+=================
+
+These are notes on how the OSUOSL is setting up OpenStack (as of May 2020).
+
+Summary
+-------
+
+We use Chef as our configuration management tool of choice and have created an environment using the `OpenStack Chef`_
+Cookbooks. The compute nodes are running CentOS 7 and the controller node is an x86 virtual machine running CentOS 7.
+All storage for the cluster is powered via a `Ceph`_ cluster backed by a 10Gb network.
```

Comment:
Something to note here, x86 & aarch64 use a 10Gb backend network while ppc uses 40Gb.